modules/core/src/main/scala/org/tessellation/http/routes/DagRoutes.scala

```diff
@@ -7,7 +7,8 @@ import cats.syntax.functor._
 import org.tessellation.dag.domain.block.L1Output
 import org.tessellation.domain.cell.{L0Cell, L0CellInput}
 import org.tessellation.domain.dag.DAGService
-import org.tessellation.ext.http4s.vars.{AddressVar, SnapshotOrdinalVar}
+import org.tessellation.ext.http4s.AddressVar
+import org.tessellation.ext.http4s.SnapshotOrdinalVar
```

Comment:
Let's hope that this will not fail the CI build ;)

Reply:
I mean formatting :)

Reply:
oh 🙈